config/metrics-server/kustomization.yaml

```diff
@@ -10,5 +10,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: ghcr.io/kedacore/keda-metrics-apiserver
-  newName: ghcr.io/kedacore/keda-metrics-apiserver
```

Comment:
Please revert this file changes

Reply:
I forget to revert the changes after installing the operator on my local cluster